apps/db_data/models.py

```diff
@@ -168,8 +169,7 @@ class Sponsorship(models.Model):
 class Event(models.Model):
     name = models.CharField(max_length=70)
     location = models.CharField(max_length=70)
-    date = models.DateField(null=True)
-    time = models.CharField(max_length=70)
+    date_time = models.DateTimeField(null=False)
```

Comment:
We need to have some way to store the start and end datetimes, not just the start